scripts/docsgen/main.go

```diff
@@ -14,18 +14,33 @@ import (
 	"golang.org/x/xerrors"
 )
 
-// This script patches Markdown docs generated by `terraform-plugin-docs` to expose the original deprecation message.
+// This script patches Markdown docs generated by `terraform-plugin-docs` to:
+// 1. Expose the original deprecation message
+// 2. Extract and format version information from resource/attribute descriptions
+// 3. Add inline version markers for attributes with version requirements
 
```

Comment:
The script should also sanitize the front matter and description from @since:vX.Y.Z mentions that only to be used to get the version, add the information to resources and attributes, not to be printed anywhere else in markdown files.